src/renderer/components/query.jsx

```diff
@@ -275,17 +286,26 @@ export default class Query extends Component {
               mode="sql"
               theme="github"
               name={this.props.editorName}
-              height="100%"
+              height="calc(100% - 15px)"
```

Comment:
wow this trick is new to me 😄

Reply:
Pretty long in the standard https://caniuse.com/#search=calc ;)